handlers/event.js

```diff
@@ -88,6 +87,73 @@ module.exports.get = async (event, ctx, callback) => {
 
 };
 
+module.exports.getUsers = async (event, ctx, callback) => {
+
+  const id = event.queryStringParameters.id;
+
+  // Check that parameters are valid
+  if (!id) {
+    callback(null, helpers.inputError('id not specified.', 'missing query param'));
+  }
+
+  const params = {
+    TableName: 'biztechRegistration' + process.env.ENVIRONMENT,
+    FilterExpression: 'eventID = :query',
+    ExpressionAttributeValues: {
+      ':query': id
+    }
+  };
+
+  await docClient.scan(params).promise()
+  .then(async result => {
+    console.log('Scan success.');
+    const registrationList = result.Items;
+
+    /**
+     * Example registration obj:
+     * { eventID: 'blueprint',
+     *   id: 123,
+     *   updatedAt: 1580007893340,
+     *   registrationStatus: 'registered'
+     * }
+     */
+    let keysForRequest = registrationList.map(registrationObj => {
+      let keyEntry = {}
+      keyEntry.id = parseInt(registrationObj.id)
+      return keyEntry
+    })
+    console.log('Keys:', keysForRequest)
+
+    let keyBatches = [];
+    const size = 100 // max BatchGetItem count
```

Comment:
the maximum number of users we can get is 100?

Reply:
Per call to `docClient.batchGet`, yes. So we have to group ids into batches of 100